packages/components-native/src/InputText/InputText.test.tsx

```diff
@@ -548,6 +549,80 @@ describe("InputText", () => {
       expect(styleOverride.inputText.color).toEqual(flattenedStyle.color);
     });
   });
+
+  describe("showMiniLabel", () => {
+    it("defaults to true", () => {
+      const props = { placeholder: "placeholder", value: "value" };
+      renderInputText(props);
+      expect(
+        screen.getByText("placeholder", { includeHiddenElements: true }),
+      ).toBeDefined();
+      expect(MockInputFieldWrapper).toHaveBeenCalledWith(
```

Comment:
same thoughts on these tests. verifying "placeholder" is there and that props were passed to a mocked component doesn't feel particularly high value as a test to me.

again, I know it was like this already - so we should look at refactoring these tests as a separate task.